src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,57 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $paddingCalendarBox;
+  width: $calendarSize;
+  box-sizing: border-box;
```

Comment:
Move box-sizing to the `*` selector and remove it in other places

Reply:
But the code of the layout_task-guideline style says that using the * selector is not advisable. Can I deviate from this rule?

Reply:
It is ok to use * for box-sizing